### ISSUE TITLE
Failed to create save file

### ISSUE THREAD

**Issue by tavich** [mroote/factorio-server-manager#98]
Ubuntu Server 16.04

There's nothing in the logs and I see it is creating the folder but it's blank.

I made a user "factorio" and ran:
`sudo ./factorio-server-manager --dir ~/.factorio
`
to start the server. Everything works up to the point of actually starting the server. The server can't start until a save file is present which isn't generating. What could be the issue?

**Comment by knoxfighter** [comment]
You need to to generate a savefile, it has two possible ways to do this:
1. Upload a savefile from your local machine
2. Generate a savefile in the factorio-server-manager
Both functionalities can be found in the Saves Tab

**Comment by tavich** [comment]
When I go to generate a save file it says "Failed to create save file"

**Comment by knoxfighter** [comment]
Then it should have some error messages in the log, would be nice, if you can post them here, so we can find whats going wrong and fix it.

**Comment by tavich** [comment]
There's nothing in the logs and I see it is creating the folder but it's blank.

**Comment by knoxfighter** [comment]
Which version of factorio and which version of the factorio-server-manager do you use?

**Comment by knoxfighter** [comment]
I cant reproduce your error, do you have any mods installed?

**Comment by tavich** [comment]
Someone good news

I was able to start a server by chmoding the ~/.factorio to 777 however it still gave a error and only if I uploaded my own save file which is what you just recommended so good timing lol.

The error looked like something was wrong with the website (something along the lines of 
`script(sucess:true,savegame:yes)` but i just ignored it. Thanks for the help. I don't know why it was giving me such a hard time.
